analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AssertionArgsShouldBePassedInCorrectOrder.MsTest.cs

```diff
@@ -11,7 +11,8 @@ public void Foo()
         {
             var str = "";
             Assert.AreEqual(str, ""); // Noncompliant {{Make sure these 2 arguments are in the correct order: expected value, actual value.}}
-//                          ^^^^^^^
+//                          ^^^
+//                               ^^   Secondary@-2
```

Comment:
@martin-strecker-sonarsource I cannot figure out the correct syntax for a primary and secondary issue location on the same line. Can you help me?